tabular/src/autogluon/tabular/task/tabular_prediction/tabular_prediction.py

```diff
@@ -277,26 +278,32 @@ def fit(train_data,
                             min_time_limit: (float, default=0) Allow this model to train for at least this long (in sec), regardless of the time limit it would otherwise be granted.
                                 If `min_time_limit >= max_time_limit`, time_limit will be set to min_time_limit.
                                 If `min_time_limit=None`, time_limit will be set to None and the model will have no training time restriction.
+                    AG_args_ensemble: Dictionary of ensemble customization options. Only models in stack levels >=1 are impacted by these parameters.
+                        Valid keys:
+                            use_orig_features: (bool) Whether a stack model will use the original features along with the stack features to train (akin to skip-connections). If the model has no stack features (no base models), this value is ignored and the stack model will use the original features.
+                            max_base_models: (int, default=25) Maximum number of base models to use when generating stack features. If more than `max_base_models` base models are available, only the top `max_base_models` models with highest validation score are used.
```

Comment:
"Maximum number of base models to use when generating stack features."
->
"Maximum number of base models whose predictions form the features input to  higher-level stacker models."

Reply:
Added